canonical-web-and-design/greenhouse-exporter
# Change workload table

## Done
- Updated `api/workload` to get scheduled interviews for the jobs that the logged user is the hiring lead
- Updated workload table in the frontend
## QA
- Log in as Peter: `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- Go to http://0.0.0.0:8501/
- Check the workload table is working
 ## Issue
#46
